### PR TITLE
Robot library remove old robots

### PR DESCRIPTION
**Pull request by yck011522**
This depends on #406, we should review and merge #406 first before this one.

Removed `compas_fab.robots.ur5` because it is now part of `compas_fab.robots.RobotLibrary`.
Removed data files of ur5 and ur10e from `src/compas_fab/data/universal_robots` because they are now in of `src/compas_fab/data/robot_library`.

Future constructs should use:
```
from compas_fab.robots import RobotLibrary
robot = RobotLibrary.ur5(load_geometry=True)
```
Or from URDF and SRDF from scratch
```
urdf_filename = compas_fab.get("robot_library/ur5_robot/urdf/robot_description.urdf")
robot_model = RobotModel.from_urdf_file(urdf_filename )

mesh_loader = LocalPackageMeshLoader(compas_fab.get("robot_library/ur5_robot"), "")
robot_model.load_geometry(mesh_loader)

srdf_filename = compas_fab.get("robot_library/ur5_robot/robot_description_semantic.srdf")
semantics = RobotSemantics.from_srdf_file(srdf_filename, robot_model)
robot = Robot(robot_model, semantics=semantics)
```

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [x] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas_fab.robots.CollisionMesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by yck011522** [comment]
@gonzalocasas (whenever you are free, not urgent). I cannot seem to get pass two integration doctests that connects to ROS in the Docker. They both threw the "Cannot Connect to ROS" error.

I see that there are only two files with docs that connects to ROS for testing. 
- `src\compas_fab\backends\ros\client.py`
- `src\compas_fab\robots\robot.py`

Somehow there was no problem connecting to ROS from the client.py file.

**Comment by yck011522** [comment]
> @gonzalocasas (whenever you are free, not urgent). I cannot seem to get pass two integration doctests that connects to ROS in the Docker. They both threw the "Cannot Connect to ROS" error.
> 
> I see that there are only two files with docs that connects to ROS for testing.
> 
> * `src\compas_fab\backends\ros\client.py`
> * `src\compas_fab\robots\robot.py`
> 
> Somehow there was no problem connecting to ROS from the client.py file.
> 
> I have tried to change the doc example to use a context manager (`with RosClient() as ros:`) but it doesn't help. I have tried adding timeout=30 to the connection but it doesn't help. I cannot reproduce the error in my local environment with python 3.8 (Windows), so maybe it is a Ubuntu thing?

Phew, Finally!! I just tried to change the Integration Test to run on py3.11 instead of 3.8 and the connection problem goes away. Can you see if you are okay with that. @gonzalocasas

**Comment by yck011522** [comment]
@gonzalocasas How do you feel about the rather frequent `testdocs` failing. Or if you have any insight why it is failing.

File: src/compas_fab/backends/ros/client.py [hunk 1]
```diff
@@ -149,6 +147,25 @@ def __enter__(self):
...
+
+        t1 = time.time()
+        wait_connect = threading.Event()
+        self.factory.on_ready(lambda _: wait_connect.set())
+
+        self.factory.manager.run()
+        if not wait_connect.wait(timeout):
+            t2 = time.time()
+            raise RosTimeoutError("Failed to connect to ROS. Start Time: {}, Time elapsed: {}".format(t1, t2 - t1))
+
```
Comment (gonzalocasas) [inline src/compas_fab/backends/ros/client.py:168]: Please delete this one! ;)

Comment (gonzalocasas) [inline src/compas_fab/backends/ros/client.py:168]:
```suggestion
```

File: src/compas_fab/robots/conftest.py [hunk 2]
```diff
@@ -27,23 +27,7 @@ def add_imports(doctest_namespace):
...
-    elif request.module.__name__ == "compas_fab.robots.planning_scene":
-        with RosClient() as client:
-            robot = Robot(client)
-
-            doctest_namespace["client"] = client
-            doctest_namespace["robot"] = robot
-
-            yield
-    else:
-        yield
```
Comment (gonzalocasas) [inline src/compas_fab/robots/conftest.py:49]: Yay! Out with the trash!

**Review by gonzalocasas — APPROVED** [review]
🚀  LGTM!

**Comment by jf---** [comment]
Amazing work @yck011522 !

**Pull request merged**